### DISCUSSION TITLE
Add a few custom icons please (lamps and sunshade)

### DISCUSSION THREAD

**Discussion by therealmcsparrow** [Mariusthvdb/custom-icons#38] [Icon suggestions/requests]
Hi I just want to add a few icons.

Tuya Desk sphered color lamp
Nordic wall lamp
Sunshade (plane/striped) (closed and open)

See attached for the SVG
![sunshade_plane](https://user-images.githubusercontent.com/46250500/184656499-e3fcbc98-c25f-46fd-8ae9-4a17f26348a5.svg)
![sunshade_plane_closed](https://user-images.githubusercontent.com/46250500/184656500-f968efd0-5ae0-4fc7-9cb0-e4cd5eaec813.svg)
![sunshade_striped](https://user-images.githubusercontent.com/46250500/184656501-6f80f7dc-085f-4c3f-99d7-8b871c4ca2f6.svg)
![sunshade_striped_closed](https://user-images.githubusercontent.com/46250500/184656502-95a3c4e2-0e5c-47bf-80da-97c4b3479e1c.svg)
![Tuya smart color desklight](https://user-images.githubusercontent.com/46250500/184656503-e3f6389e-10d5-4a1b-94c9-59e1ec8eba47.svg)
![Nordic wall light](https://user-images.githubusercontent.com/46250500/184657171-2e448725-fd9a-4286-bf50-f4a1ffac5036.svg)

**Comment by Mariusthvdb** [comment]
Hi,

Before I can check this (on vacation...) did you set the size correctly? they hardly conform to any icons sizing used in HA?
Also: what is the resource for these icons? Did you create them? If not, you'd need to check with the owner if they're ok with adding this

**Reply by therealmcsparrow** [reply]
Hi Marius, 

Yes, they were made by me. 
I forked your repository to check them in HA,
I am recreating them, because how I created them is not showing correctly in HA. I have to simplify them. I will repost the end result and I will make a RP.

Have a nice holiday, cheers

Carlo

**Comment by therealmcsparrow** [comment]
Hi Marius,

Created a pull request and added the svg files in docs\svg

Cheers

Carlo
